packages/cli/src/PublicApi/index.ts

```diff
@@ -33,21 +31,26 @@ function createApiRouter(
 	];
 	const apiController = express.Router();
 
-	apiController.use(
-		`/${publicApiEndpoint}/${version}/docs`,
-		swaggerUi.serveFiles(swaggerDocument),
-		swaggerUi.setup(swaggerDocument, {
-			customCss: swaggerThemeCss,
-			customSiteTitle: 'n8n Public API UI',
-			customfavIcon: `${n8nPath}favicon.ico`,
-		}),
-	);
+	if (!config.getEnv('publicApi.swaggerUi.disabled')) {
+		const { serveFiles, setup } = await import('swagger-ui-express');
```

Comment:
when this is disabled, we should also update the message and playground link in `SettingsApiView.vue`, linking to static docs hosted on https://docs.n8n.io/api/api-reference/

Reply:
done